.github/workflows/deployment-ec2.yml

```diff
@@ -0,0 +1,27 @@
+name: Deploy on aws EC2
```

Comment:
I think it's time to rename the file to be explicit that this script is for deploying the web server only and not the mobile application. What do you think about "deploy-web-ec2.yml" ?

Reply:
I'll go for "deploy-web-app-ec2.yml", thanks 🙂